src/algebra/add_torsor.lean

```diff
@@ -401,6 +409,32 @@ variable {P}
 
 open function
 
+/-- Given an additive automorphism `f` of `G`, the operation `p ↦ f (p -ᵥ p₀) +ᵥ p₀`, "basing" the
+additive automorphism at a fixed point `p₀ : P`, is an affine equivalence. -/
+def _root_.add_equiv.base_at (f : G ≃+ G) (x : P) : perm P :=
+((vadd_const x).symm.trans f.to_equiv).trans (vadd_const x)
```

Comment:
A few other reasonable definitions:
```suggestion
def _root_.equiv.base_at {G' P' : Type*} [add_group G'] [add_torsor G' P'] (f : G ≃ G') (x : P) (y : P') :
  P ≃ P':=
((vadd_const x).symm.trans f).trans (vadd_const y)

def _root_.add_equiv.base_at {G' P' : Type*} [add_group G'] [add_torsor G' P'] (f : G ≃+ G') (x : P) (y : P') :
  P ≃ P':=
f.to_equiv.base_at x y

def _root_.add_aut.base_at (f : add_aut G) (x : P) :
  perm P :=
f.base_at x x
```

Reply:
You're right that one could introduce a more general `base_at` construction which constructs, from an isomorphism of model spaces, an isomorphism of the `add_torsor`s according to a fixed basepoint `x`, `y` of each.  It would double the size of the PR, though, so I'd rather not add it until someone needs it for something.  (The version with a single `add_torsor`, constructing an automorphism, is more immediately useful -- it allows one to talk sensibly to talk about rotations in affine space.)

On your other point, I'd be happy to change the definition here to
```suggestion
def _root_.equiv.base_at (f : G ≃ G) (x : P) : perm P :=
((vadd_const x).symm.trans f).trans (vadd_const x)
```
if you prefer, or include both.

Reply:
@urkud :ping_pong: